cmd/erasure-object.go

```diff
@@ -368,8 +370,51 @@ func (er erasureObjects) getObject(ctx context.Context, bucket, object string, s
 	return er.getObjectWithFileInfo(ctx, bucket, object, startOffset, length, writer, fi, metaArr, onlineDisks)
 }
 
+func (er erasureObjects) GetObjectDebugInfo(ctx context.Context, bucket, object string, opts ObjectOptions, prefix string) (debugInfo map[string]string) {
+	lk := er.NewNSLock(bucket, object)
+	if err := lk.GetRLock(ctx, globalOperationTimeout); err != nil {
+		return nil
+	}
+	defer lk.RUnlock()
+
+	disks := er.getDisks()
+	debugInfo = make(map[string]string)
```

Comment:
Please make this structured an not just a bunch of string -> string.

Reply:
The intent here is to pass through raw information all the way to `mc` and not tying it to specific API response. This is by design.